ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/queries/imageDetails.ts

```diff
@@ -0,0 +1,50 @@
+import { gql } from '@apollo/client';
+
+export type ImageDetailsVariables = {
+    id: string;
+};
+
+export type ImageDetailsResponse = {
+    image: {
+        deploymentCount: number;
+        name: {
+            fullName: string;
+        } | null;
+        operatingSystem: string;
+        metadata: {
+            v1: {
+                created: Date | null;
```

Comment:
what's the difference between typing it this way vs ` created?: Date` ?

Reply:
`created?: Date` would allow either `{ created: undefined }` or `{}` in the response object, while GraphQL will explicitly add a `null` for nullable fields `{ created: null }`. In practice it probably doesn't matter a whole lot since we are checking for truthy values in the code, but I figured I'd keep it as accurate as possible.